### PR TITLE
Maximum Weight

### PR DESCRIPTION
**Pull request by Udumft**
THis PR adds `RouteOptions.maximumWeight` property.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@
 * Types that correspond to objects in the Mapbox Directions API response, such as `RouteResponse`, `RouteRefreshResponse`, `MatchResponse`, and `RouteStep`, now conform to the `ForeignMemberContainer` and `ForeignMemberClassContainer` protocols. Types that conform to these protocols can persist unrecognized properties in the response, such as properties that are in beta, even after coding and decoding. You can access these properties using the `ForeignMemberContainer.foreignMembers` and `ForeignMemberClassContainer.foreignMembers` properties. ([#669](https://github.com/mapbox/mapbox-directions-swift/pull/669))
 * Fixed an issue where decoding a `RouteResponse` incorrectly set the `Waypoint.snappedDistance` property to `nil`. ([#669](https://github.com/mapbox/mapbox-directions-swift/pull/669)) 
 * The `mapbox-directions-swift` command line tool now requests routes from the Mapbox Directions API if no input file is specified. ([#576](https://github.com/mapbox/mapbox-directions-swift/pull/576))
+* Added `RouteOptions.maximumWeight` property to compute routes that includes only roads with appropriate weight limit. ([#694](https://github.com/mapbox/mapbox-directions-swift/pull/694))
```
Comment (1ec5) [inline CHANGELOG.md:10]:
```suggestion
* Added the `RouteOptions.maximumWeight` property to compute routes that excludes roads with a lower weight limit. ([#694](https://github.com/mapbox/mapbox-directions-swift/pull/694))
```

Comment (1ec5) [reply CHANGELOG.md:10]: The next prerelease of this library will be the final release candidate for v2.5.0. If this PR doesn’t make that prerelease, then remember to move this entry up to a new v2.6.0 section.

**Review by 1ec5 — APPROVED** [review]
Looks good to me. Please hold this PR until the `beta` label is removed from the `max_weight` parameter in the Directions API documentation.

**Comment by Udumft** [comment]
Documentation page has been updated, PR is unblocked to merge now

**Pull request merged**